pkgs/tools/cd-dvd/cuetools/default.nix

```diff
@@ -15,6 +15,10 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ bison flac flex id3v2 vorbis-tools ];
 
+  postInstall = ''
```

Comment:
Maybe a comment here that it's for compat with other distros?

Reply:
added, thanks for the suggestion!